### PR TITLE
BLADE-728 update blade sample for gradle 7

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: liferay-workspace-test-integration-sample/settings.gradle [hunk 1]
```diff
@@ -1,7 +1,11 @@
 buildscript {
 	dependencies {
-		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "4.1.11"
-		classpath group: "net.saliman", name: "gradle-properties-plugin", version: "1.4.6"
+		classpath group: "org.yaml", name: "snakeyaml", version: "1.32"
+		classpath group: "biz.aQute.bnd", name: "biz.aQute.bnd", version: "5.3.0"
+		classpath group: "com.github.docker-java", name: "docker-java-transport", version: "3.2.8"
+		classpath(group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "8.0.8") {
+			exclude group: "biz.aQute.bnd", module: "biz.aQute.bnd"
+		}
```
Comment (drewbrokke) [inline liferay-workspace-test-integration-sample/settings.gradle]: Can we remove some of these dependencies from here now that we are on v8.0.8 or workspace?

File: liferay-workspace-test-integration-sample/modules/sample/sample-test/build.gradle [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+	testIntegrationImplementation group: "junit", name: "junit", version: "4.13.1"
+	testIntegrationImplementation group: "org.apache.logging.log4j", name: "log4j-api", version: "2.18.0"
+	testIntegrationImplementation group: "org.apache.logging.log4j", name: "log4j-core", version: "2.18.0"
+	testIntegrationImplementation group: "org.slf4j", name: "log4j-over-slf4j", version: "1.7.25"
+	testIntegrationImplementation project(":modules:sample:sample-api")
+	testIntegrationImplementation project(":modules:sample:sample-service")
 
-	testModules group: "com.liferay", name: "com.liferay.arquillian.extension.junit.bridge.connector"
-	testModules group: "com.liferay.portal", name: "com.liferay.portal.test", version: "latest.release"
-	testModules group: "com.liferay.portal", name: "com.liferay.portal.test.integration", version: "latest.release"
```
Comment (drewbrokke) [inline liferay-workspace-test-integration-sample/modules/sample/sample-test/build.gradle]: Is changing the versions from `latest.release` related to updating to Gradle 7? If not we should make this change separately.

File: liferay-workspace-test-integration-sample/gradle.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 liferay.workspace.modules.dir = modules
 liferay.workspace.themes.dir = themes
 liferay.workspace.wars.dir = modules
-liferay.workspace.product = portal-7.4-ga86
+liferay.workspace.product = portal-7.4-ga85
```
Comment (drewbrokke) [inline liferay-workspace-test-integration-sample/gradle.properties]: Do we need to decrement this version?

File: liferay-workspace/wars/simple-theme/src/main/webapp/WEB-INF/liferay-plugin-package.properties [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 change-log=
-licenses=Apache
-liferay-versions=7.0.0+,7.1.0+,7.2.0+,7.3.0+
+licenses=LGPL
+liferay-versions=7.0.0+,7.1.0+,7.2.0+,7.3.0+,7.4.0+
 long-description=
 module-group-id=liferay
 module-incremental-version=1
-name=blade.theme
+name=untitled2333333333333
```
Comment (drewbrokke) [inline liferay-workspace/wars/simple-theme/src/main/webapp/WEB-INF/liferay-plugin-package.properties]: This doesn't seem right - shouldn't this be something like "simple-theme"?

File: yarn-workspace/settings.gradle [hunk 5]
```diff
@@ -1,16 +1,21 @@
 buildscript {
 	dependencies {
-		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "latest.release"
-		classpath group: "net.saliman", name: "gradle-properties-plugin", version: "1.4.6"
+		classpath group: "org.yaml", name: "snakeyaml", version: "1.32"
+
+		classpath group: "biz.aQute.bnd", name: "biz.aQute.bnd", version: "5.3.0"
+		classpath group: "com.github.docker-java", name: "docker-java-transport", version: "3.2.8"
+		classpath(group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "8.0.3") {
```
Comment (drewbrokke) [inline yarn-workspace/settings.gradle]: In some places we are using `8.0.8`, in others `latest.release`, and here it's `8.0.3`. Is there a particular reason for this version here?

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@simonjhy thanks for sending this. I had a couple of comments, please see my inline comments. Thanks!

File: liferay-workspace-test-integration-sample/settings.gradle (hunk 1 above)
Comment (simonjhy) [reply liferay-workspace-test-integration-sample/settings.gradle]: OK

File: liferay-workspace-test-integration-sample/modules/sample/sample-test/build.gradle (hunk 2 above)
Comment (simonjhy) [reply liferay-workspace-test-integration-sample/modules/sample/sample-test/build.gradle]: I revert this change

File: yarn-workspace/settings.gradle (hunk 5 above)
Comment (simonjhy) [reply yarn-workspace/settings.gradle]: will use the same versoin 8.0.8 for workspace plugin

File: liferay-workspace/wars/simple-theme/src/main/webapp/WEB-INF/liferay-plugin-package.properties (hunk 4 above)
Comment (simonjhy) [reply liferay-workspace/wars/simple-theme/src/main/webapp/WEB-INF/liferay-plugin-package.properties]: ok

File: liferay-workspace-test-integration-sample/gradle.properties (hunk 3 above)
Comment (simonjhy) [reply liferay-workspace-test-integration-sample/gradle.properties]: change to use portal-7.4-ga94

**Pull request closed without merging**